source/brailleDisplayDrivers/handyTech.py

```diff
@@ -24,7 +24,25 @@
 import bdDetect
 import time
 import datetime
+from ctypes import windll
+import windowUtils
 from driverHandler import BooleanDriverSetting
+import wx
+
+class InvisibleDriverWindow(windowUtils.CustomWindow):
+	className = u"Handy_Tech_Server"
+	def __init__(self, driver):
+		super(InvisibleDriverWindow, self).__init__(u"Handy Tech Server")
+		self.driver = driver

```

Comment:
Please make this a weak reference. You can see how this works on the Model class.

There is also a possibility to provide a callback to the reference that is called when the reference dies, i.e. when the driver instance dies. We really need to make sure that the window is destroyed in that case. Could you also test this, e.g. by forcefully disconnecting a device and checking whether the window is destroyed correctly? I assume it will if terminate is called when the driver loses its connection.